api/admin.py

```diff
@@ -24,7 +24,16 @@
 from notifications.models import RecordType, SubscriptionType
 
 
+class ProfileInline(admin.StackedInline):
+    model = models.Profile
+    can_delete = False
+    verbose_name_plural = 'User Profile'
```

Comment:
@GregoryHorvath Let's use lazy translation for verbose names.
```suggestion
    verbose_name_plural = _('user profile')
```

Note: https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#lazy-translation

> Just like you would do in verbose_name you should provide a lowercase verbose name text for the relation as Django will automatically titlecase it when required.